docs/classes/ERC725.md

```diff
@@ -640,33 +690,52 @@ ERC725.encodeKeyName(
 erc725.isValidSignature(messageOrHash, signature);
 ```
 
-Checks if a signature was signed by the `owner` of the ERC725 Account contract, according to [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271). If the `owner` is a contract itself, it will delegate the `isValidsignature()` call to the owner contract, if it supports [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271), otherwise it will fail.
+The function checks if a signature was signed by the `owner` of the ERC725 Account contract, according to [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271). If the `owner` is a contract itself, it will delegate the `isValidsignature()` call to the owner contract if it supports [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271). Otherwise, the function will fail.
 
 #### Parameters
 
-1. `messageOrHash` - `string`: If the string is not a hash (66 chars long with 0x prefix), it will be enveloped as follows: `"\x19Ethereum Signed Message:\n" + message.length + message` and hashed using keccak256.
-2. `signature` - `string`: The raw RLP encoded signature (can be generated with [`web3.eth.accounts.sign()`](https://web3js.readthedocs.io/en/v1.2.11/web3-eth-accounts.html#sign)).
+| Name            | Type   | Description                                           |
+| :-------------- | :----- | :---------------------------------------------------- |
+| `messageOrHash` | string | value of an message or hash that needs to be verified |
+| `signature`     | string | The raw RLP encoded signature                         |
+
+:::info
+
+- The hash must be 66 chars long with the `0x` prefix, otherwise it will be interpreded as message.
+- The message will be: enveloped as `"\x19Ethereum Signed Message:\n" + message.length + message` <br/> and hashed using `keccak256` function.
```

Comment:
why the line break here?

Reply:
The code format splits up between 2 lines and does not look convenient. I wanted to state that there are two things going on: one line signing, one line hashing